iset.mm

```diff
@@ -57231,6 +57241,60 @@ ordered pairs (for use in defining operations).  This is a special case
       $}
     $}
 
+    ${
+      caofid0.3 $e |- ( ph -> B e. W ) $.
+      ${
+        caofid0l.5 $e |- ( ( ph /\ x e. S ) -> ( B R x ) = x ) $.
+        $( Transfer a left identity law to the function operation.
+           (Contributed by NM, 21-Oct-2014.) $)
+        caofid0l $p |- ( ph -> ( ( A X. { B } ) oF R F ) = F ) $=
+          ( vw cv cfv csn wcel wceq co cxp wfn fnconstg ffnd fvconst2g sylan wa
+          syl eqidd wral ralrimiva ffvelcdmda oveq2 id eqeq12d rspccva syl2an2r
+          offveq ) ANCDNOZGPZECDQUAZGGHJADIRZVACUBLCDIUCUHACFGKUDZVCAVBUSCRZUSV
+          APDSLCDUSIUEUFAVDUGUTUIADBOZETZVESZBFUJVDUTFRDUTETZUTSZAVGBFMUKACFUSG
+          KULVGVIBUTFVEUTSZVFVHVEUTVEUTDEUMVJUNUOUPUQUR $.
+      $}
+
+      ${
+        caofid0r.5 $e |- ( ( ph /\ x e. S ) -> ( x R B ) = x ) $.
+        $( Transfer a right identity law to the function operation.
+           (Contributed by NM, 21-Oct-2014.) $)
+        caofid0r $p |- ( ph -> ( F oF R ( A X. { B } ) ) = F ) $=
+          ( vw cv cfv csn wcel wceq co cxp ffnd wfn fnconstg wa eqidd fvconst2g
+          syl sylan wral ralrimiva ffvelcdmda oveq1 id eqeq12d rspccva syl2an2r
+          offveq ) ANCNOZGPZDEGCDQUAZGHJACFGKUBZADIRZVACUCLCDIUDUHVBAUSCRZUEUTU
+          FAVCVDUSVAPDSLCDUSIUGUIABOZDETZVESZBFUJVDUTFRUTDETZUTSZAVGBFMUKACFUSG
+          KULVGVIBUTFVEUTSZVFVHVEUTVEUTDEUMVJUNUOUPUQUR $.
+      $}
+
+      caofid1.4 $e |- ( ph -> C e. X ) $.
+      ${
+        $d C w x $.
+        caofid1.5 $e |- ( ( ph /\ x e. S ) -> ( x R B ) = C ) $.
+        $( Transfer a right absorption law to the function operation.
```

Comment:
right => left? We should use the labels ~caoflid and ~caofrid as we do in similar cases.